presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java

```diff
@@ -187,6 +190,44 @@ public void shouldGetTableTypes()
         assertThat(result).contains(row("TABLE"), row("VIEW"));
     }
 
+    @Test(groups = {JDBC, SIMBA_JDBC})
+    public void testSqlEscapeFunctions()
+            throws SQLException
+    {
+        if (usingSimbaJdbcDriver()) {
+            // These functions, which are implemented within the Simba JDBC and ODBC drivers,
+            // translate standard SQL syntax to equivalent Presto syntax.  The translated SQL
```

Comment:
I think it would be great if we added support for this syntax to Presto, but I was not involved in that decision.  I suspect it was done this way to save time, but maybe @mattsfuller or @rschlussel could explain why we had it put in the driver instead of adding it to Presto.